spinalcordtoolbox/reports/qc2.py

```diff
@@ -273,6 +274,116 @@ def sct_register_multimodal(
         fig.savefig(img_path, format='png', transparent=True, dpi=300)
 
 
+def sct_deepseg(
+    fname_input: str,
+    fname_seg: str,
+    species: str,
+    argv: Sequence[str],
+    path_qc: str,
+    dataset: Optional[str],
+    subject: Optional[str],
+    fname_seg2: Optional[str],
+):
+    """
+    Generate a QC report for sct_deepseg, with varied colormaps depending on the type of segmentation.
+
+    This refactor is based off of the `listed_seg` method in qc.py, adapted to support multiple images.
+    """
+    command = 'sct_deepseg'
+    cmdline = [command]
+    cmdline.extend(argv)
+
+    # Axial orientation, switch between one anat image and 1-2 seg images
+    with create_qc_entry(
+        path_input=Path(fname_input).absolute(),
+        path_qc=Path(path_qc),
+        command=command,
+        cmdline=list2cmdline(cmdline),
+        plane='Axial',
+        dataset=dataset,
+        subject=subject,
+    ) as imgs_to_generate:
+        # FIXME: This code is more or less duplicated with the 'sct_register_multimodal' report, because both reports
+        #        use the old qc.py method "_make_QC_image_for_3d_volumes" for generating the background img.
+        # Resample images slice by slice
+        p_resample = {
+            'human': 0.6, 'mouse': 0.1,
+        }[species]
```

Comment:
Just to demonstrate what happens when I try to create a QC report for mouse data using the human-centric 0.6mm resampling...

Before:

![image](https://github.com/spinalcordtoolbox/spinalcordtoolbox/assets/16181459/1704485b-066f-4787-a1ad-4c89b885cdde)

After:

![image](https://github.com/spinalcordtoolbox/spinalcordtoolbox/assets/16181459/5503c855-9cc2-4679-8abd-093959070de3)

(Ignore the poor QC quality, that has to do with the input data.)